docs/release-notes/release-notes-0.18.0.md

```diff
@@ -133,6 +133,10 @@
 * [Add a watchtower tower client
   multiplexer](https://github.com/lightningnetwork/lnd/pull/7702) to manage
   tower clients of different types.
+* 
+* [Introduce CommitmentType and JusticeKit
+  interface](https://github.com/lightningnetwork/lnd/pull/7736) to simplify the
```

Comment:
Ah only realized there was a stacked PR after getting to this point, can fix w/e comments on that one.